src/core/MRApp.js

```diff
@@ -389,7 +389,23 @@ export class MRApp extends MRElement {
         }
 
         this.camera.matrixWorldAutoUpdate = false;
-        this.camera.position.set(0, 0, 1);
+
+        const startPosString = this.getAttribute('userPos');
```

Comment:
i went with the `userPos="x y z"` approach since it's the user's starting position